include/BandLimitedWave.h

```diff
@@ -101,6 +101,7 @@ class LMMS_EXPORT BandLimitedWave
 	BandLimitedWave() = default;
 	virtual ~BandLimitedWave() = default;
 
+	static const float DIVISION_FLOOR = 1e-4; // This can be changed
```

Comment:
We'll need to figure out a good small number here. I'm not sure if 0.0001 is suitable.

Reply:
If we're dividing by a frequency, we could make it as high as 1